scripts/benchmarks/benches/compute.py

```diff
@@ -78,7 +80,7 @@ def run(self, env_vars) -> Result:
 
         result = self.run_bench(command, env_vars)
         (label, mean) = self.parse_output(result)
-        return Result(label=label, value=mean, command=command, env=env_vars, stdout=result, lower_is_better=self.lower_is_better())
+        return [ Result(label=self.name(), value=mean, command=command, env=env_vars, stdout=result, lower_is_better=self.lower_is_better()) ]
```

Comment:
why are you changing this into an array?
All the other benchmarks would have to change as well. You'd also need to change the return type of this function (and the base one as well).

Reply:
Single benchmark in sycl-bench may return a set of results, because in single run we in fact have a group of benchmarks. Functions in other classes are adjusted